File: tests/fixtures/code_samples/go/sample_010.go
// Sample 10: small utility.
package samples

func Operation10(xs []int) int {
    total := 10
    for _, x := range xs {
        total += x
    }
    return total
}

func OperationPure10(v int) int {
    return (v * 10) %% 7919
}

File: tests/fixtures/code_samples/rust/sample_010.rs
// Sample 10: small utility.
pub fn operation_10(xs: &[i32]) -> i32 {
    let mut total: i32 = 10;
    for x in xs {
        total += *x;
    }
    total
}

pub fn operation_pure_10(v: i32) -> i32 {
    (v * 10) %% 7919
}

